.circleci/config.yml

```diff
@@ -57,79 +54,91 @@ jobs:
     steps:
       - prep
       - build
-#        - test
 
   build_deploy_pypi:
     docker:
       - image: circleci/python:3.8
     steps:
       - checkout
       - run:
-          name: Install dependencies
-          command: |
-            sudo apt-get update
-            sudo apt-get install build-essential
-            pip install setuptools twine
-      - run:
-          name: Build Python package
+          name: Check if release commit and deploy to PyPI
           command: |
-            python setup.py sdist
-      - run:
-          name: Upload Python package
-          command: |
-            twine upload dist/* --username __token__ --password $PYPI_CRED
+            if [[ $(git log -1 --pretty=%B) == "chore(main): release"* ]]; then
+              sudo apt-get update
+              sudo apt-get install build-essential
+              pip install setuptools twine
+              python setup.py sdist
+              twine upload dist/* --username __token__ --password $PYPI_CRED
+            else
+              echo "Skipping PyPI deployment as this is not a release commit"
+            fi
       - store_artifacts:
           path: dist
 
+  release-pr:
+    docker:
+      - image: cimg/node:20.2.0
+    steps:
+      - checkout
+      - run: npm ci
+      - run: |
+          npx release-please release-pr \
+            --token $GITHUB_TOKEN \
+            --repo-url $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME \
+            --target-branch $CIRCLE_BRANCH \
+            --config-file .circleci/release/release-please-config.json \
+            --manifest-file .circleci/release/release-please-manifest.json
```

Comment:
you have a file in this commit named 
`.circleci/release/.release-please-manifest.json`

but these commands are looking for 
`.circleci/release/release-please-manifest.json`

should remove the leading . from the filename

Reply:
thank you

Reply:
putting the `.` in the command works too i guess